hedera-node/src/main/java/com/hedera/services/txns/schedule/ScheduleReadyForExecution.java

```diff
@@ -37,16 +39,19 @@
 	}
 
 	ResponseCodeEnum processExecution(ScheduleID id) throws InvalidProtocolBufferException {
+		var executionStatus = store.markAsExecuted(id);
```

Comment:
I prefer to see "final" in situations like this. It makes reading the code easier, because as I read the code (from top to bottom) I can tell just from this line that `executionStatus` isn't going to be modified anywhere further down. As it is, I'm not sure, so I have to read the rest of the code in the block scope to verify that it is "effectively final".

Reply:
Made the changes.